lib/qb_integration/address.rb

```diff
@@ -4,6 +4,7 @@ def self.build(order_address)
       address = Quickbooks::Model::PhysicalAddress.new
       address.line1   = hash_or_empty(order_address, 'address1')
       address.line2   = hash_or_empty(order_address, 'address2')
+      address.line3 = hash_or_empty(order_address, 'address3')
```

Comment:
But the alignment...!

![imagen](https://user-images.githubusercontent.com/662596/75690850-52084900-5c82-11ea-87f2-2a4c59017077.png)

Reply:
Lol, i'll update that right now